infra/cifuzz/actions/entrypoint.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Builds and runs specific OSS-Fuzz project's fuzzers for CI tools."""
+
+import os
+import subprocess
+
+
+def main():
+  """Runs OSS-Fuzz project's fuzzers for CI tools."""
+  project_name = os.environ['OSS_FUZZ_PROJECT_NAME']
+  repo_name = os.environ['GITHUB_REPOSITORY'].rsplit('/', 1)[-1]
+  commit_sha = os.environ['GITHUB_SHA']
+
+  # Build the specified project's fuzzers from the current repo state
+  print('Building fuzzers\nproject: %s\nrepo name: %s\nbranch: %s\ncommit: %s' %
+        (project_name, repo_name, commit_sha))
+  command = [
+      'python3', '/src/oss-fuzz/infra/cifuzz/cifuzz.py', 'build_fuzzers',
+      project_name, repo_name, commit_sha
+  ]
+  print('Running command: %s' % command)
+  try:
+    subprocess.check_call(command, stdout=stdout, stderr=subprocess.STDOUT)
+  except subprocess.CalledProcessError as e:
+    print('Error building fuzzers.')
```

Comment:
`print('Error building fuzzers: "{0}"'.format(str(e)))`

Reply:
also, for errors, it's better to write them to `stderr`, so instead of `print` you might want to use `sys.stderr.write` and the string argument would need a trailing `'\n`, i.e.:

```python
sys.stderr.write('Error building fuzzers: "{0}"\n'.format(str(e)))
```

Reply:
Done.